README.md

```diff
@@ -131,6 +131,15 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [PepsiCo](https://www.pepsicojobs.com/internshipsprograms/jobs/241240?lang=en-us&previousLocale=en-US)| Purchase, New York <br> Plano, Texas | Software Development & Engineering Intern (No Sponsorship) |
 | [U.S. Bank](https://careers.usbank.com/global/en/job/2023-0020491/2024-Software-Engineering-Summer-Intern-Early-Careers) | Richfield, MN; and Atlanta, GA | 2024 Software Engineering Summer Intern |
 | Jane Street | New York, NY | [Software Engineer Internship](https://www.janestreet.com/join-jane-street/position/6834728002/) <br/> [Quantitative Trader Internship](https://www.janestreet.com/join-jane-street/position/6850721002/) <br/> [Quantitative Researcher Internship](https://www.janestreet.com/join-jane-street/position/6838982002/) |
+| [SkyWorks](https://careers.skyworksinc.com/job/Austin-Embedded-FW-Summer-Intern-2024-TX-73301/1054957600) | Austin, TX | Embedded FW Summer Intern |
+| [Cole Engineering Services, Inc.](https://workforcenow.adp.com/mascsr/default/mdf/recruitment/recruitment.html?cid=5055dea5-b670-4a19-adb1-2aea04ba5207&ccId=19000101_000001&type=MP&lang=en_US&jobId=472432) | Orlando, FL | Intern (Software) (US Citizenship Required) |
```

Comment:
> Please note that pursuant to a government contract, this specific position requires U. S. Citizenship status with ability to obtain a SECRET to TOP SECRET security clearance.  Security Clearance requirements will be specified in the Government's Task Order.

"Will need security clearance" isn't really something we make a note of, but it's probably important enough to note here. Would like to get some other opinions, though.

Reply:
>  with ability to obtain a SECRET to TOP SECRET security clearance

If you're a US citizen, you have the ability to obtain a security clearance. No problem here.